public/main.css

```diff
@@ -0,0 +1,198 @@
+.input-range-container {
```

Comment:
We need to cleanup this file somewhere down the line

Reply:
probably should do it sooner than later, would rather keep it nice than deal with css spaghetti 😄